Analyzers/ValidateSpeakerTopics.cs

```diff
@@ -66,7 +66,7 @@ protected override void AnalyzeSpeakerData(IDocument document, IAnalyzerContext
             IReadOnlyList<string> topics = document.GetList<string>(SiteKeys.Topics);
             if (topics == null || topics.Count == 0)
             {
-                context.Add(document, "No topics specified");
+                // No topics, but that's okay (if we ever want to enforce at least one topic, add an analyzer result here)
```

Comment:
Turns out we're getting some speakers without topics, so I figure it's better to let those pass validation instead of fail the build.